### PR TITLE
[Draft/TEST]Add android testing script

### PR DESCRIPTION
**Pull request by struss**
For #4702 

update build script for install(binary output).
update options for buliding for cmake.
add android testing script.

ONE-DCO-1.0-Signed-off-by: KiDeuk Bang <rrstrous@nate.com>

### CODE REVIEW

**Comment by struss** [comment]
Any comments are welcome!

File: infra/scripts/test_android_runtime_release.sh [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+if [ -z "$SERIAL" ]; then
+    SERIAL=`echo "$DEVICE_LIST" | tail -n1 | awk '{print $1}'`
+fi
+ADB_CMD="adb -s $SERIAL "
+
+# root on, remount as rw
+$ADB_CMD root on
+$ADB_CMD shell mount -o rw,remount /
+
+$ADB_CMD shell rm -rf /data/local/tmp
```
Comment (chunseoklee) [inline infra/scripts/test_android_runtime_release.sh:44]: This will /data/local/tmp/ folder ? This folder may contain pre installed binaries by users.

Comment (struss) [reply infra/scripts/test_android_runtime_release.sh:44]: Oh, I don't know that... then I think I should change the binary installation folder on others. Thanks for pointing out!

**Comment by hseok-oh** [comment]
Outdated

**Pull request closed without merging**